Inquartata trait

Apocarypse opened the pull request:
<!-- place 'x' mark between square [] brackets to affirm: -->
**_I affirm:_**
- [x] that I agree to Project Topaz's [Limited Contributor License Agreement](http://project-topaz.com/blob/release/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I've _tested my code_ since the last commit in the PR, and will test after any later commits

Added mod enum for Inquartata in modifier.h
Added entries for Inquartata in traits.sql
Applied Inquartata bonus logic to GetParryRate()

Apocarypse committed:
Inquartata trait
Added mod enum for Inquartata in modifier.h
Added entries for Inquartata in traits.sql
Applied Inquartata bonus logic to GetParryRate()

Kreidos reviewed (approved):
Looks good to me!

ibm2431 commented on src/map/utils/battleutils.cpp:
```diff
@@ -1723,6 +1723,15 @@ namespace battleutils
                 }
 
+                // Inquartata grants a flat parry rate bonus.
+                if (CCharEntity* PChar = dynamic_cast<CCharEntity*>(PDefender))
```
Sverdheid and Zurko-Bazurko are RUN mobs that players need to fight.

Kreidos replied:
I think based on what was said on the discord, that ibm's idea was that if you aren't needing the trait check, then there's no need to cast to CCharEntity*. You can just use PDefender directly as CBattleEntity has the getMod method. :)

Apocarypse committed:
Removed the trait check
inquartataBonus should be giving 0 any time
it's called and entity doesn't have the trait.

Apocarypse committed:
Dropped ccharentity entirely

TeoTwawki commented:
@Apocarypse @cocosolos @ibm2431 modifiers in modifier.h and statsu.lua should mirror each other until such time as lua can automagically use the core enums, even if the mod is never used in script so that future editor don't mistakenly use the same ID - this has happened multiple times in the older project history until I became its watchdog in dsp.

ibm2431 commented:
_sees this merged PR in notifications_
_sees Teo as a participant_

"I _did_ miss one, didn't I?"

_clicks_

"Yeeeep. Whoops."
